src/Formatter/RemoveSensitiveQueryStringsFormatter.php

```diff
@@ -33,9 +33,25 @@ public function formatRequest(RequestInterface $request): string
         return $this->removeQueryStrings($request);
     }
 
+    /** @psalm-suppress DeprecatedMethod */
     public function formatResponse(ResponseInterface $response): string
     {
-        return $this->formatter->formatResponse($response);
+        return sprintf(
```

Comment:
any reason why this implementation changed? Doesnt seem right?

Reply:
Fixed it